### PR TITLE
Changed focus to select in InputText and LabeledInput components.

### LINKED ISSUE #68: Add select option for InputText and LabeledInput components
Value in input needs to be selected.

### PR DESCRIPTION
**Pull request by oskarwrobel**
Fixes #68

### CODE REVIEW

**Comment by oleq** [comment]
Are you sure we should drop the `focus` interface? It will surely make sense in the future.

**Comment by oskarwrobel** [comment]
I thought about it, and assumed that we will add it when will be necessary.

File: src/inputtext/inputtextview.js [hunk 1]
```diff
@@ -46,10 +46,10 @@ export default class InputTextView extends View {
 	}
 
 	/**
-	 * Sets focus to the input.
+	 * Moves the focus to the input and select entire value.
```
Comment (oleq) [inline src/inputtext/inputtextview.js]: s/and select entire value/and selects the value

File: src/labeledinput/labeledinputview.js [hunk 2]
```diff
@@ -29,9 +29,9 @@ export default class LabeledInputView extends View {
 	}
 
 	/**
-	 * Moves the focus to the input.
+	 * Moves the focus to the input and select entire value.
```
Comment (oleq) [inline src/labeledinput/labeledinputview.js]: s/and select entire value/and selects the value

**Comment by oleq** [comment]
Alright. So fixes to the docs only and R+.

**Pull request merged**